doc/nrf/index.rst

```diff
@@ -3,6 +3,16 @@
 Welcome to the |NCS|!
 #####################
 
+.. important::
+
+   |NCS| v1.9.2-dev1 is a development tag and will be replaced by a release in the future.
+
+   The development tag contains the following updates:
+
+   * Added workaround for the anomaly 160 that is affecting the nRF5340 SoC.
```

Comment:
Do we need a bulleted list? 
```suggestion
   The development tag contains the addition of workaround for the anomaly 160 that is affecting the nRF5340 SoC.
```
or 

```suggestion
   The development tag adds a workaround for the anomaly 160 that is affecting the nRF5340 SoC.
```

Reply:
Yes, I'm still waiting for the input from the Bluetooth team :)